src/Azure.Functions.Cli/Helpers/VersionHelper.cs

```diff
@@ -53,13 +53,13 @@ public static async Task<bool> IsRunningAnOlderVersion()
                 {
                     var jProperty = (Newtonsoft.Json.Linq.JProperty)item;
                     var releaseDetail = JsonConvert.DeserializeObject<ReleaseDetail>(jProperty.Value.ToString());
-                    releaseList.Add(new ReleaseSummary() { Release = jProperty.Name, ReleaseDetail = releaseDetail.ReleaseList.FirstOrDefault() });
+                    releaseList.Add(new ReleaseSummary(jProperty.Name, releaseDetail.ReleaseList.FirstOrDefault()));
                 }
 
-                var latestCoreToolsReleaseVersion = releaseList.FirstOrDefault(x => x.Release == data.Tags.V4Release.ReleaseVersion)?.CoreToolsReleaseNumber;
+                var latestCoreToolsAssemblyZipFile = releaseList.FirstOrDefault(x => x.Release == data.Tags.V4Release.ReleaseVersion)?.CoreToolsAssemblyZipFile;
 
-                if (!string.IsNullOrEmpty(latestCoreToolsReleaseVersion) &&
-                    Constants.CliVersion != latestCoreToolsReleaseVersion)
+                if (!string.IsNullOrEmpty(latestCoreToolsAssemblyZipFile) &&
```

Comment:
Can we add a test for `IsRunningAnOlderVersion`?

Reply:
@kshyju Added the test scripts for the above. could you review once.